src/userdel.c

```diff
@@ -827,7 +832,19 @@ static int remove_mailbox (void)
 	if (NULL == maildir) {
 		return 0;
 	}
-	snprintf (mailfile, sizeof mailfile, "%s/%s", maildir, user_name);
+
+	len = strlen (prefix) + strlen (maildir) + strlen (user_name) + 2;
```

Comment:
It feels like a very long prefix could risk overflowing the stack in the alloca() below.  Do you think it's worth a sanity check?

Reply:
The max path length is limited in most systems. But indeed, I will add either a check on the max path or change the alloca() call to malloc()/free() (if possible) (but I think this is not specific to this part of the code. I do remember having seen alloca() in a few places)